collab/src/lib/parse-steps.ts

```diff
@@ -0,0 +1,26 @@
+import type { Json } from "../types/json";
+import { stepJsonIds, type StepModel } from "../types/step";
+
+/* eslint-disable -- linter complains of @typescript-eslint/strict-boolean-expressions on line 7. Tried to disable that specific rule but it claims it isn't enabled */
+const isStep = (item: Json): item is StepModel => {
+  return !!(
```

Comment:
Think we don't need the !! here

Reply:
We still don't need this `!!` I think? The part in brackets will always evaluate to a boolean. `!!` is to coerce types to boolean.